web-components/src/components/modal/Modal.ts

```diff
@@ -272,8 +272,8 @@ export namespace Modal {
                   id="${this.htmlId}"
                   class="md-modal ${classMap(this.modalContainerClassMap)}"
                   aria-label="${this.ariaLabel}"
-                  aria-labelledby="modal_header"
-                  aria-describedby=${ifDefined(this.ariaDescription)}
+                  aria-labelledby=${ifDefined(this.ariaLabelledBy || undefined)}
```

Comment:
why are we removing "modal_header" from aria-lablledby as heading needs to be linked to it or can we keep it as default if this needs to be made dynamic ?

Reply:
Hi, this was introduced by me as a fix for this story only. We do not need this now. So keeping it dynamic only as this.ariaLabelledBy.
Prev PR - https://github.com/momentum-design/momentum-ui/pull/1555/files

Reply:
The problem I see with this change is it resulted in fixing multiple modal not just station credential modal, you might need to verify other modals if their behavior is not impacted with this change.

Reply:
updated the initial value. It won't impact other modal behaviour now.